content/guides/autodiscovery.md

```diff
@@ -48,155 +83,253 @@ By default, the Datadog Agent includes Autodiscovery support for:
 - Redis
 - Riak
 
-These are provided by the configuration templates in the Datadog Agent `conf.d/auto_conf` directory.
+If you want to monitor Apache containers, for example, the default Apache template may not suit you if:
 
-To add Autodiscovery for your custom container images, you simply need to add a configuration template to the `conf.d/auto_conf` directory.
+1. You want to use options not used in the default template,
+1. the Apache containers run a different image than that specified in the default template (see the example below), OR
+1. the Apache containers expose more than one port (the default template doesn't use [template variable indexing](#template-variable-indexes)).
 
-## Configuration templates
+You can provide template files of your own in a few ways:
 
-The configuration templates in `conf.d/auto_conf` directory are nearly identical to the example YAML configuration files provided in [the Datadog `conf.d` directory](https://github.com/DataDog/dd-agent/tree/master/conf.d), but with one important field added. The `docker_images` field is required and identifies the container image(s) to which the configuration template should be applied.
+1. Add them to each host that runs docker-dd-agent and [mount the directory that contains them](https://github.com/DataDog/docker-dd-agent#configuration-files) into the docker-dd-agent container when starting it
+1. Package them into your own release of docker-dd-agent
```

Comment:
I'd go with:

Build a custom docker image based on docker-dd-agent with your custom templates added in the `/etc/dd-agent/conf.d/auto_conf`